### PR TITLE
Update README

### PR DESCRIPTION
**Pull request by alexandreroutier**
Hi,

Please find the final proposition for the Readme file: https://github.com/alexandreroutier/clinica/tree/readme

Of note, images are hosted on the server of Clinica.

I can add an extra commit to move `clinica_logo.png` created by Katja onto this server @mdiazmel .

Best,
Alex

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,15 +1,123 @@
...
+<h1 align="center">
+  <a href="http://www.clinica.run">
+    <img src="http://www.clinica.run/assets/images/clinica-icon-257x257.png" alt="Logo" width="120" height="120">
+  </a>
+  <br/>
+  Clinica
+</h1>
 
-[**Clinica**](http://www.clinica.run) is a software platform for clinical neuroimaging studies.
+<p align="center"><strong>Software platform for clinical neuroimaging studies.</strong></p>
```
Comment (nburgos) [inline README.md:11]: I would remove the dot.

File: README.md [hunk 2]
```diff
@@ -1,15 +1,123 @@
...
+
+
+
+## About The Project
+
+Clinica is a software platform for clinical research studies involving patients with neurological and psychiatric diseases and the acquisition of multimodal data (neuroimaging, clinical and cognitive evaluations, genetics...), most often with longitudinal follow-up.
+
+Clinica is command-line driven and written in Python. It uses the [Nipype](https://nipype.readthedocs.io/) system for pipelining and combines widely-used software packages for neuroimaging data analysis ([ANTs](http://stnava.github.io/ANTs/), [FreeSurfer](https://surfer.nmr.mgh.harvard.edu/), [FSL](https://fsl.fmrib.ox.ac.uk/fsl/fslwiki), [MRtrix](https://www.mrtrix.org/), [PETPVC](https://github.com/UCL/PETPVC), [SPM](https://www.fil.ion.ucl.ac.uk/spm/)), machine learning ([Scikit-learn](https://scikit-learn.org/stable/)) and the [BIDS standard](http://bids-specification.readthedocs.io/) for data organization.
+
+Clinica provides tools to convert publicly available neuroimaging datasets into BIDS namely:
```
Comment (nburgos) [inline README.md:50]: Add comma before "namely".

**Comment by alexandreroutier** [comment]
Final idea: What about adding a "Related Repositories" section below "License" section e.g.:

## Related Repositories

- [Framework for the reproducible classification of Alzheimer's disease using deep learning](https://github.com/aramis-lab/AD-DL)
- [Framework for the reproducible classification of Alzheimer's disease using machine learning](https://github.com/aramis-lab/AD-ML)

😇

**Comment by mdiazmel** [comment]
Thanks @alexandreroutier . I'll merge this one.

**Pull request merged**